api.yml

```diff
@@ -773,10 +773,16 @@ paths:
                     x-stoplight:
                       id: 7h1cpgfjj5j9j
                     format: date-time
+                  uptime:
+                    type: string
+                    type: number
+                    format: int32
+                    minimum: 0
```

Comment:
I am really not sure about that. Please tell me what to put if it needs a change ;-)

Reply:
Would just be the same as in `Status.yaml` 

```yaml
                  uptime:
                    type: integer
                    description: EVSE gateway uptime, in seconds
```